Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 max_time = 180
 
 setup_mxnet_gpu = """
-    python3 -m pip install mxnet-cu102==1.8.*
+    python3 -m pip install mxnet-cu101==1.7.*
```

Comment:
1) this mxnet version have outstanding CVEs opened against it. We should upgrade to 1.9.x 
2) cu101 might be an issue with containers: our tests showed 10.2 is needed.

Reply:
cu101 is the only one I got it to work with the CI. will check again re mxnet version (this is coupled to the GluonTS requirement).

Reply:
After a lot of experimentation, I was able to bump the mxnet in CI to 1.9, and make sure cuda 10.2 with cuDNN support was installed (via conda). This effects `timeseries` unit tests as well as tutorial build jobs. Note that the prod code does not include any constraints on the cuda version, and leaves the cuda/mxnet version up to the user.